projects/plugins/boost/tests/e2e/lib/pages/wp-admin/JetpackBoostPage.js

```diff
@@ -18,7 +18,7 @@ export default class JetpackBoostPage extends WpPage {
 	async chooseFreePlan() {
 		const button = this.page.locator( 'text=Start for free' );
 		await button.click();
-		await this.waitForElementToBeVisible( '.jb-section--scores' );
+		await this.waitForElementToBeVisible( '[data-testid="speed-scores"]' );
```

Comment:
🤔 Well, it's a solution that works around the generated module class names. I think it makes sense. 👍